runtime/doc/lua-guide.txt

```diff
@@ -93,10 +93,11 @@ Finally, you can include Lua code in a Vimscript file by putting it inside a
 Using Lua files on startup                                    *lua-guide-config*
 
 Nvim supports using `init.vim` or `init.lua` as the configuration file, but
-not both at the same time. This should be placed in your |config| directory,
-which is typically `~/.config/nvim` for Linux, BSD, or macOS, and
-`~/AppData/Local/nvim/` for Windows. Note that you can use Lua in `init.vim`
-and Vimscript in `init.lua`, which will be covered below.
+not both at the same time. This should be placed in your |config| directory
+(run `:echo stdpath('config')` to see where it is). Note that you can also use
+Lua in `init.vim` and Vimscript in `init.lua`, which will be covered below.
+
+Run `:e $VIMRUNTIME/example_init.lua` to see an example of a Lua config file.
```

Comment:
probably want a one-liner codeblock to copy the config to the right location. similar to the steps from `:help nvim-from-vim`.

and this would probably go in `:help nvim-quickstart`